### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brandonrc**
Added updates for latest 2.4ghz hardware support and added a range test for the 2.4ghz DIY module.

### CODE REVIEW

**Comment by SpencerGraffunder** [comment]
Suggestions:
remove the word "support" in line 24.
it's unclear in the 'rf noise' section whether other 900MHz RC control systems will interfere with 900MHz ELRS.
since the hardware compatibility table has 2.4ghz stuff already, the 'Where can I get 2.4GHz if I cannot solder?' seems redundant.

**Comment by brandonrc** [comment]
@SpencerGraffunder I think one big issue I have seen on Facebook and even in our discord is this misconception that you have to build your own 2.4ghz stuff. 

Like the news has not gotten out that we have so many other options than DIY 2.4 ghz TX's... so many may be a stretch but yea. 

I have to constantly respond and help educate people and they end up saying why is the github README so out of date 🤷‍♂️ 

@AlessandroAU some feed back on the Where can I get 2.4GHz if I cannot solder? Have you also noticed people say there are no options for 2.4 outside of building your own?

File: README.md [hunk 1]
```diff
@@ -54,8 +69,15 @@ Development is ongoing but the following hardware is currently compatible
...
-| RX        | Ghost Zepto     | Fully Supported     | Can only be flashed via stlink               |
+| RX        | Ghost Atto      | Fully Supported     | Can only be initially flashed via stlink then betaflight passthrough |
+| RX        | Ghost Zepto     | Fully Supported     | Can only be initially flashed via stlink then betaflight passthrough |
+| TX        | Ghost TX        | Fully Supported     | Can only be initialy flashed via stlink (Development Branch)| 
+
+**Siyi Hardware**
+| **RX/TX** | **Hardware**    | **Status**          | **Notes**                                    |
+| --------- | --------------- | ------------------- | -------------------------------------------- |
+| TX | FM30 | Fully Supported | Stlink initial flash, USB update |
+|RX | FM30 Mini | Development | In STlink initial flash, betaflight passthrough update
```
Comment (cruwaller) [inline README.md:80]: Add space before `RX` and `|` to EOL.

File: README.md [hunk 2]
```diff
@@ -21,7 +21,22 @@ ExpressLRS can be flashed into existing **Frsky R9M hardware (RX and TX)**, **Ju
...
+ExpressLRS aims too achieve the best possible link preformance for both latency and range. This is achieved with an optimised over the air packet structure.  However, only basic telemetry is currently provided (**VBAT**, downlink/uplink **LQ** and downlink/uplink **RSSI**), full telemetry support is supported in our development branch. This comprimise allows ExpressLRS to achieve simultaneous **better latency AND range** compared to other options in the market. For example, **ExpressLRS 2.4GHz 150Hz** mode offers the same range as **GHST Normal** while delivering near **triple** the packet update rate. Similarly, **ExpressLRS 900MHz 200Hz** will dramatically out-range **Crossfire 150Hz** and **ExpressLRS 50Hz** will out-range **Crossfire 50Hz** watt per watt.   
+
+## Should I use 2.4ghz or 915/868mhz?
```
Comment (cruwaller) [inline README.md:26]: Align casing ( `ghz -> GHz` and `mhz` to `MHz`) and use it also later (not only 2.4 :) )

Comment (cruwaller) [reply README.md:26]: And `hz` to `Hz`

**Review by cruwaller — CHANGES_REQUESTED** [review]

**Comment by CapnBry** [comment]
PR is obsolete now and I've integrated some of this text into the wiki. Closing after discussion with Brandon.

**Pull request closed without merging**